plenum/test/txn_author_agreement/helper.py

```diff
@@ -1,11 +1,26 @@
+from typing import NamedTuple, Dict
+
 from indy.ledger import build_txn_author_agreement_request
 
-from plenum.common.constants import CONFIG_LEDGER_ID
+from plenum.common.constants import (
+    CONFIG_LEDGER_ID,
+    TXN_AUTHOR_AGREEMENT_TEXT, TXN_AUTHOR_AGREEMENT_VERSION,
+    TXN_PAYLOAD, TXN_METADATA, TXN_METADATA_SEQ_NO, TXN_METADATA_TIME
+)
+from plenum.common.util import randomString
 from plenum.server.config_req_handler import ConfigReqHandler
 from plenum.test.helper import sdk_sign_and_submit_req, sdk_get_and_check_replies
 
 
-def sdk_send_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet, text: str, version: str):
+TaaData = NamedTuple("TaaData", [
+    ("version", str),
+    ("text", str),
+    ("seqNo", int),
+    ("txnTime", int)
+])
+
+
+def sdk_send_txn_author_agreement(looper, sdk_pool_handle, sdk_wallet, version: str, text: str):
```

Comment:
One more comment - I also like version before text more, however I think this should follow SDK more closely, which have these parameters as text, then version. Also keep in mind that this function is already used in indy-node, so merging this PR with parameters reversed will probably break node tests.

Reply:
unified as `text, version` sequence